server/lib/service/dv360.dart

```diff
@@ -31,6 +31,36 @@ class DisplayVideo360Client {
         updateMask: 'entityStatus');
   }
 
+  /// Changes the bidding strategy of the line item to [strategy].
+  ///
+  /// Throws an [ApiRequestError] if API returns an error.
+  Future<void> changeLineItemBiddingStrategy(
+      Int64 advertiserId, Int64 lineItemId, String strategy,
+      {Int64 bidAmount, String goal, Int64 goalAmount}) async {
+    final request = LineItem()..bidStrategy = BiddingStrategy();
+    switch (strategy) {
+      case 'FIXED':
+        request.bidStrategy.fixedBid =
+            (FixedBidStrategy()..bidAmountMicros = bidAmount.toString());
+        break;
+      case 'MAXIMIZE_SPEND':
+        request.bidStrategy.maximizeSpendAutoBid =
+            (MaximizeSpendBidStrategy()..performanceGoalType = goal);
+        break;
+      case 'PERFORMANCE_GOAL':
+        request.bidStrategy.performanceGoalAutoBid =
+            (PerformanceGoalBidStrategy()
+              ..performanceGoalType = goal
+              ..performanceGoalAmountMicros = goalAmount.toString());
+        break;
+      default:
+        throw UnimplementedError('$strategy is not a supported strategy.');
```

Comment:
If there are not more strategy to be added, maybe use UnsupportedError like other places? Unless you plan to add more cases in here.

Reply:
That makes sense! I don't plan on adding more strategies here.